Remote clusters missing use cases and structural changes (#4170)
elastic/docs-content: deploy-manage/remote-clusters/eck-remote-clusters.md

  - id: cloud-kubernetes
---

# Connect to {{es}} clusters in the same ECK environment [k8s-remote-clusters-connect-internal]

> These steps describe how to configure remote clusters between two {{es}} clusters that are managed by the same {{eck}} (ECK) operator. Once that’s done, you’ll be able to [run CCS queries from {{es}}](/solutions/search/cross-cluster-search.md) or [set up CCR](/deploy-manage/tools/cross-cluster-replication/set-up-cross-cluster-replication.md). For other remote cluster scenarios with ECK, refer to [Remote clusters on ECK](./eck-remote-clusters-landing.md).

Reviewer: ```suggestion
These steps describe how to configure remote clusters between two {{es}} clusters that are managed by the same {{eck}} (ECK) operator. 

After the connection is established, you’ll be able to [run CCS queries from {{es}}](/solutions/search/cross-cluster-search.md) or [set up CCR](/deploy-manage/tools/cross-cluster-replication/set-up-cross-cluster-replication.md). For other remote cluster scenarios with ECK, refer to [](./eck-remote-clusters-landing.md).
```
